h/security/permission_map.py

```diff
@@ -50,7 +50,9 @@
     ],
     Permission.Group.EDIT: [
         [p.group_matches_authenticated_client_authority],
-        [p.group_created_by_user],
+        [p.group_has_user_as_owner],
+        [p.group_has_user_as_admin],
+        [p.group_has_user_as_moderator],
```

Comment:
This means that the group's creator is no longer permitted to edit the group, and instead the group's owners, admins and moderators are.